ground/gcs/src/plugins/config/configinputwidget.cpp

```diff
@@ -1784,3 +1774,64 @@ void ConfigInputWidget::checkHangtimeConfig()
 
     m_config->lblHangTimeWarning->setVisible(warn);
 }
+
+void ConfigInputWidget::clearMessages(QWidget *widget, const QString type)
+{
+    QLabel *lbl = findChild<QLabel *>("lblMessage" + type);
+    if (lbl) {
+        widget->layout()->removeWidget(lbl);
+        delete lbl;
+    }
+}
+
+void ConfigInputWidget::addMessage(QWidget *widget, const QString type, const QString msg)
+{
+    QLabel *lbl = findChild<QLabel *>("lblMessage" + type);
+    if (!lbl) {
+        lbl = new QLabel(this);
+        lbl->setObjectName("lblMessage" + type);
+        lbl->setWordWrap(true);
+        widget->layout()->addWidget(lbl);
```

Comment:
Should probably not assume the widget has a layout, rather add one if it doesn't before using it.